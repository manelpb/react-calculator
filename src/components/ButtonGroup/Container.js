import styled from 'styled-components';

const Container = styled.div`
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  max-width: ${props => props.width || '154px'};
`;

export default Container;