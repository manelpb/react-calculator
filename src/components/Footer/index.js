import React, { PureComponent } from 'react';
import Container from 'components/Container';

class Footer extends PureComponent {
  render() {
    return (
      <Container className="App">
      </Container>
    );
  }
}

export default Footer;
