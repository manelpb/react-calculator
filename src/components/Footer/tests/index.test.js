import React from 'react';
import Footer from '../';
import Container from 'components/Container';
import { shallow } from 'enzyme';

it('should render a Container', () => {
    const render = shallow(<Footer />);
    expect(render.find(Container).length).toEqual(1);
    expect(render).toMatchSnapshot();
})