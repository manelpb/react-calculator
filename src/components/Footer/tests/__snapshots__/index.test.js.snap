// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Container 1`] = `
<styled.div
  className="App"
/>
`;
